### ISSUE TITLE
check_http does not properly handle intermediate certificate expiration

### ISSUE THREAD

**Issue by jazzl0ver** [nagios-plugins/nagios-plugins#627]
```
[opc@centreon ~]$ /usr/lib/nagios/plugins/check_http -V
check_http v2.3.3 (nagios-plugins 2.3.3)
[opc@centreon ~]$ /usr/lib/nagios/plugins/check_http -H dev.domain.com -S -C 10 --verify-host
SSL OK - Certificate 'dev.domain.com' will expire in 20 days on 2021-10-21 08:00 +0000/GMT.
```
Here is ssllabs.com test result:
![image](https://user-images.githubusercontent.com/2444175/135466439-72d0fb1b-a0da-4a9d-8d19-fb0bc1926b12.png)

**Comment by sawolf** [comment]
Hi, thanks for reaching out. I definitely agree that the plugin would be more useful if it was able to check additional certificates.

That said, I don't have a server like this on hand. Does the browser give warnings/refuse to load the page when the intermediate certificate is expired?

**Comment by jazzl0ver** [comment]
No, it appears the browsers implemented some kind of fix for that. I had to spent a lot of time trying to figure out why our app is not working, while cert monitoring showed no issues. Even after I re-issued certificates the problem didn't disappear. The final tweak was to update ca-bundle.txt on our servers by manually deleting DST Root CA X3 certificate.
